src/uwtools/cli.py

```diff
@@ -49,6 +50,7 @@ def main() -> None:
     try:
         setup_logging(quiet=True)
         args, checks = _parse_args(sys.argv[1:])
+        args[STR.action] = args[STR.action] if STR.action in args else args[STR.mode]
```

Comment:
I'm afraid not...

Reply:
Is it bc there's no `get` method on a Namespace object?

Reply:
I want to circle back to this, but maybe we're missing something in `cli.py` because `uw -h` does't show the `invoke` mode. I think we had this issue with another mode once and agreed that integration / regression tests will help catch this. Unless I'm doing something wrong, I think we need to mention `STR.invoke` in another place to get it wired into the top-level CLI.

Reply:
I updated the suggestion above. The default value shouldn't be `STR.mode`, but `args[STR.mode]`, I think.

Reply:
Paul, I changed the name from `invoke` to `execute` and I see `execute` listed when I use `uw -h`.

Reply:
🤦 of course. So fast though! Nice.